src/components/workers-page/workers-tab/enhanced-table-header-component.tsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+import TableCell from "@material-ui/core/TableCell";
+import TableSortLabel from "@material-ui/core/TableSortLabel";
+import { Button } from "../../common-components";
+import { WorkerData } from "../../../common-models/worker-info.model";
+import { Order } from "../../../helpers/array.helper";
+import { createStyles, makeStyles } from "@material-ui/core/styles";
+import ScssVars from "../../../assets/styles/styles/custom/_variables.module.scss";
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    label: {
+      color: ScssVars.primary,
+      fontWeight: "normal",
+      fontSize: ScssVars.fontSizeBase,
+      fontFamily: ScssVars.fontFamilyPrimary,
+      letterSpacing: ScssVars.headingLetterSpacing,
```

Comment:
To mi się nie podoba, bo sporo powtórzyłem kodu. Jeszcze to zrefactoruję :)

Reply:
ja nie widzę już tu powtórzeń